src/model/encoding/convertFromHTMLToContentBlocks.js

```diff
@@ -116,31 +118,78 @@ function getBlockDividerChunk(block: DraftBlockType, depth: number): Chunk {
   };
 }
 
-function getBlockTypeForTag(tag: string, lastList: ?string): DraftBlockType {
-  switch (tag) {
-    case 'h1':
-      return 'header-one';
-    case 'h2':
-      return 'header-two';
-    case 'h3':
-      return 'header-three';
-    case 'h4':
-      return 'header-four';
-    case 'h5':
-      return 'header-five';
-    case 'h6':
-      return 'header-six';
-    case 'li':
-      if (lastList === 'ol') {
-        return 'ordered-list-item';
-      }
-      return 'unordered-list-item';
-    case 'blockquote':
-      return 'blockquote';
-    case 'pre':
-      return 'code-block';
-    default:
+function getListBlockType(
+  tag: string,
+  lastList: ?string
+): ?DraftBlockType {
+  if (tag === 'li') {
+    if (lastList === 'ol') {
+      return 'ordered-list-item';
+    }
+    return 'unordered-list-item';
+  }
+
+  return null;
+}
+
+function getBlockMapSupportedTags(
+  draftBlockRenderMap: DraftBlockRenderMap
+): Array<string> {
```

Comment:
as discussed on slack the problem is because 

```
271 function containsSemanticBlockMarkup(
272   html: string,
273   blockTags: Array<DraftBlockTag>
274 ): boolean {
275   return blockTags.some(tag => html.indexOf('<' + tag) !== -1);
276 }
```

the string comparison of indexOf makes flow thing that the type for blockTags needs to be a string

so it fails as:

```
src/model/encoding/convertFromHTMLToContentBlocks.js:292
292:   blockTags: Array<string>,
                       ^^^^^^ string. This type is incompatible with
138: :disappointed: Array<DraftBlockTag> {
             ^^^^^^^^^^^^^ string enum
```

We decided then to leave as string type for now